run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+PATH_TO_PROJECT=$(pwd)
+export PKG_CONFIG_PATH=$PATH_TO_PROJECT/rust-wrapper/target/x86_64-unknown-linux-gnu/debug
+export LD_LIBRARY_PATH=$PATH_TO_PROJECT/rust-wrapper/target/x86_64-unknown-linux-gnu/debug
+cd $PATH_TO_PROJECT/rust-wrapper
+cbindgen --lang=c -o haskell-wrapper/cbits/rust-wrapper.h
+cargo cbuild
```

Comment:
I get an error "no such command" on this line.

Reply:
You need to install cargoc package with:
```bash
cargo install cargo-c
```
It's important that the version of cargo is 1.80.0. I recommend installing via rustup